rss_repository.py

```
@@ -0,0 +1,55 @@
+import sqlite3
+from datetime import datetime, timedelta
+
+class RSSRepository:
+    def __init__(self, db_file):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)